packages/ses-ava/src/ses-ava-test.js

```diff
@@ -100,27 +145,26 @@ const wrapTester = (testerFunc, logger = defaultLogger) => {
  * Then the calls to `test` in the rest of the test file will act like they
  * used to, except that, if a test fails because the test function (the
  * callback argument to `test`) throws or returns a promise
- * that eventually rejects, the error is first sent to the `console` before
- * propagating into `rawTest`.
+ * that eventually rejects, the error is first sent to the logger
+ * (which defaults to using the SES-aware `console.error`)
+ * before propagating into `rawTest`.
  *
- * @param {TesterInterface} avaTest
+ * @template {import('ava').TestFn} T ava `test`
+ * @param {T} avaTest
  * @param {Logger} [logger]
- * @returns {TesterInterface}
+ * @returns {T}
  */
 const wrapTest = (avaTest, logger = defaultLogger) => {
-  const testerWrapper = /** @type {TesterInterface} */ (
-    wrapTester(avaTest, logger)
-  );
-  for (const methodName of testerMethodsWhitelist) {
-    if (methodName in avaTest) {
-      /** @type {TesterFunc} */
-      const testerMethod = (title, implFunc, ...otherArgs) =>
-        avaTest[methodName](title, implFunc, ...otherArgs);
-      testerWrapper[methodName] = wrapTester(testerMethod);
-    }
+  const sesAvaTest = augmentLogging(avaTest, logger);
+  for (const methodName of overrideList) {
+    sesAvaTest[methodName] = augmentLogging(avaTest[methodName], logger);
```

Comment:
While this assignment is my mistake and fixing it is not relevant to the purpose of this PR, please fix it anyway as a drive by, while you can still fit it into this PR. We should generally define properties using define semantics rather assignment semantics.